release/trigger-build

```diff
@@ -1 +1 @@
-1.10.0-alpha.0
```

Comment:
Why a jump from 1.10 to 1.12 ?
Did we not follow the normal process before?

Reply:
Yeah we forgot to do a 1.11 alpha